domains/alarios.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "Angel Lars portfolio",
+    "repo": "https://github.com/lars505/alarios",
+    "owner": {
+      "username": "Lars505",
+      "email": "lars50593@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://lars505.github.io/"
```

```suggestion
      "CNAME": "lars505.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).